src/tests/cipher.cpp

```diff
@@ -118,13 +118,12 @@ TEST_F(rnp_tests, pkcs1_rsa_test_success)
     uint8_t ptext[1024 / 8] = {'a', 'b', 'c', 0};
     uint8_t dec[1024 / 8];
 
-    rnp_keygen_crypto_params_t key_desc;
-    key_desc.key_alg = PGP_PKA_RSA;
-    key_desc.hash_alg = PGP_HASH_SHA256;
-    key_desc.rsa.modulus_bit_len = 1024;
-    key_desc.ctx = &global_ctx;
+    rnp::KeygenParams keygen(PGP_PKA_RSA, global_ctx);
+    auto &            rsa = dynamic_cast<pgp::RSAKeyParams &>(keygen.key_params());
```

Comment:
weird spacing:)

Reply:
Yeah, unfortunately that's how clang-format 11 behaves. Don't like these as well.